src/Skeleton/ScoreSkeleton.qml

```diff
@@ -61,7 +73,7 @@ Item {
     /*
     // TODO : Creating the volume slider object
     ScoreVolume {
-        id: scoreVolume
+        id: scoreVolumeS
```

Comment:
why the S in capital ? isn't that a typo or cat that walked on the keyboard ?

Reply:
it's especially suspicious considering that nothing else in this commit seems to refer to this id

Reply:
I think it is because of a wrong ctrl-s ctrl-r :)